src/table/resizer/index.tsx

```diff
@@ -238,7 +240,9 @@ export function Resizer({
         tabIndex={tabIndex}
         data-focus-id={focusId}
       />
-      <ScreenreaderOnly id={resizerWidthId}>{headerCellWidthString}</ScreenreaderOnly>
+      <Portal container={getDescriptionRoot?.()}>
```

Comment:
Is Lighthouse happy about rendering an extra portal for every table column?

Why do you need that much complexity here? Can you think if a simpler solution?

Reply:
Thanks, I will run the tests to make sure.

I haven't found a much simpler solution so far. The portal can be made simpler if removing the container but that makes the a11y tools unhappy because of the extra nodes added that are not wrapped in any landmarks.

Another idea was to only add the description when the resize handle is in focus. However, that has still the potential to affect the tests.

Reply:
I need to look into the labelling logic overall, because I am not sure you implemented it correctly in the first place: https://github.com/cloudscape-design/components/pull/1492

This may need revisiting

Reply:
But Lighthouse test run need to happen first.

Reply:
The lighthouse tests are green.